client-js/app/elements/labrad-registry.ts

```diff
@@ -182,8 +182,8 @@ export class LabradRegistry extends polymer.Base {
   updateKey(event) {
     console.log('iron-form-submit', event);
     var self = this;
-    var selKey = Object.keys(event.detail)[0];
-    var newVal = event.detail[selKey];
+    var selKey = event.detail[0];
+    var newVal = event.detail[1];
```

Comment:
you can use destructuring assignment in typescript:
```typescript
var [selKey, newVal] = event.detail;
```
or, if you make `event.detail` an object as suggested above:
```typescript
var {key: selKey, value: newVal} = event.detail;
```
of course in the latter case, you can just refer to `event.detail.key` and `event.detail.value` directly.